values.yaml

```diff
@@ -13,6 +13,10 @@ fullnameOverride: ""
 replicaCount: 3
 daemonSet: false
 
+test:
```

Comment:
Good point.

Do you have something particular in mind for a comment here?

I was thinking:

```
# This will enable the test-ready Pod (https://github.com/datawire/ambassador-chart/blob/master/templates/tests/test-ready.yaml).
# It will spawn a busybox container to call Ambassador's check_ready endpoint to validate it is working correctly.
```

Reply:
LGTM!

Reply:
Added the comment.